src/handlers/motions/helpers.ts

```diff
@@ -9,17 +9,34 @@ export const getMotionSide = (vote: BigNumber): MotionSide =>
   vote.eq(MotionVote.YAY) ? MotionSide.YAY : MotionSide.NAY;
 
 export const updateMotionInDB = async (
-  id: string,
-  motionData: MotionData,
+  motionData: ColonyMotion,
   showInActionsList?: boolean,
 ): Promise<void> => {
-  await mutate('updateColonyAction', {
+  await mutate('updateColonyMotion', {
     input: {
-      id,
-      motionData,
-      ...(showInActionsList === undefined ? {} : { showInActionsList }),
+      ...motionData,
     },
   });
+
+  if (showInActionsList !== undefined) {
+    const { items: colonyAction } =
```

Comment:
Small detail, but renaming `items` to `colonyAction` kind of hides the fact that it's an array, and not a single object